src/functions/template-tags/day.php

```diff
@@ -1,51 +1,38 @@
 <?php
-/**
- * Single Day Test
- *
- * Returns true if the query is set for single day, false otherwise
- *
- * @category Events
- * @return bool
- */
-function tribe_is_day() {
-	$tribe_ecp = Tribe__Events__Main::instance();
-	$is_day    = ( $tribe_ecp->displaying == 'day' ) ? true : false;
+if ( ! function_exists( 'tribe_is_day' ) ) {
+	/**
+	 * Single Day Test
+	 *
+	 * Returns true if the query is set for single day, false otherwise
+	 *
+	 * @category Events
+	 * @return bool
+	 */
+	function tribe_is_day() {
+		$view_slug = tribe_context()->get( 'view', 'default' );
 
-	return apply_filters( 'tribe_is_day', $is_day );
-}
+		if ( 'default' === $view_slug ) {
+			$view_slug = tribe( \Tribe\Events\Views\V2\Manager::class )->get_default_view_slug();
+		}
 
-/**
- * Link Event Day
- *
- * @category Events
- * @param string $date
- *
- * @return string URL
- */
-function tribe_get_day_link( $date = null ) {
-	$tribe_ecp = Tribe__Events__Main::instance();
+		$is_day = 'day' === $view_slug;
 
-	return apply_filters( 'tribe_get_day_link', $tribe_ecp->getLink( 'day', $date ), $date );
+		return apply_filters( 'tribe_is_day', $is_day );
+	}
 }
 
-/**
- * Get the next day's date
- *
- * @category Events
- *
- * @return string
- */
-function tribe_get_next_day_date() {
-	return null;
-}
+if ( ! function_exists( 'tribe_get_day_link' ) ) {
+	/**
+	 * Link Event Day
+	 *
+	 * @category Events
+	 * @param string $date
+	 *
+	 * @return string URL
+	 */
+	function tribe_get_day_link( $date = null ) {
+		$tribe_ecp = Tribe__Events__Main::instance();
 
-/**
- * Get the previous day's date
- *
- * @category Events
- *
- * @return string
- */
-function tribe_get_previous_day_date( ) {
-	return null;
-}
+		return apply_filters( 'tribe_get_day_link', $tribe_ecp->getLink( 'day', $date ), $date );
```

Comment:
This is the V1  way of getting a link `tribe_events_get_url`

Reply:
Update it to the v2 way and push those changes.